### PR TITLE
add logic of creating default workspace for new project

### PR DESCRIPTION
**Pull request by BelehovEgor**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/641

### CODE REVIEW

File: src/MessageService.Models.Db/DbChannel.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.MessageService.Models.Db
+{
+    public class DbChannel
+    {
+        public const string TableName = "Channels";
+
+		public Guid Id { get; set; }
+		public Guid WorkspaceId { get; set; }
```
Comment (MariyaSin) [inline src/MessageService.Models.Db/DbChannel.cs:13]: remove empty spases

File: src/MessageService.Models.Db/DbChannel.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        public Guid OwnerId { get; set; }
+        public Guid? ImageId { get; set; }
+        public string Name { get; set; }
+        public bool IsActive { get; set; }
+        public bool IsPrivate { get; set; }
+        public DateTime CreatedAt { get; set; }
+        public DateTime? DeactivatedAt { get; set; }
+
+
```
Comment (MariyaSin) [inline src/MessageService.Models.Db/DbChannel.cs:22]: remove empty line

File: src/MessageService.Models.Db/DbWorkspaceUsers.cs [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+        public Guid Id { get; set; }
+        public Guid UserId { get; set; }
+        public Guid WorkspaceId { get; set; }
+        public bool IsAdmin { get; set; }
+        public bool IsActive { get; set; }
+        public DateTime CreatedAt { get; set; }
+        public DateTime? DeactivatedAt { get; set; }
+
+
```
Comment (MariyaSin) [inline src/MessageService.Models.Db/DbWorkspaceUsers.cs:20]: remove empty line

**Review by MariyaSin — APPROVED** [review]

File: src/MessageService.Models.Db/DbWorkspaceUsers.cs (hunk 3 above)
Comment (BelehovEgor) [reply src/MessageService.Models.Db/DbWorkspaceUsers.cs:20]: fixed

File: src/MessageService.Models.Db/DbChannel.cs (hunk 2 above)
Comment (BelehovEgor) [reply src/MessageService.Models.Db/DbChannel.cs:22]: fixed

**Review by djino97 — APPROVED** [review]

**Review by TIove — APPROVED** [review]

**Review by borovHokage — APPROVED** [review]

**Pull request merged**